<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ticket_Sort_Manage</class>
 <widget class="QDialog" name="Ticket_Sort_Manage">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>774</width>
    <height>386</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>票种添加</string>
  </property>
  <widget class="QLabel" name="ticket_name_label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>110</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>输入票种名称</string>
   </property>
  </widget>
  <widget class="QLabel" name="ticket_price_label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>150</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>输入票种进价</string>
   </property>
  </widget>
  <widget class="QPushButton" name="confirm">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="ticket_price_label_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>190</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>输入票种售价</string>
   </property>
  </widget>
  <widget class="QLabel" name="ticket_price_label_3">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>150</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>选择计价方式</string>
   </property>
  </widget>
  <widget class="QComboBox" name="purchase_compute_way">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>150</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="ticket_price_label_4">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>190</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>选择计价方式</string>
   </property>
  </widget>
  <widget class="QComboBox" name="sell_compute_way">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>190</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="select_date_btn">
   <property name="geometry">
    <rect>
     <x>74</x>
     <y>70</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>选择添加日期</string>
   </property>
  </widget>
  <widget class="QListView" name="add_detail_list">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>50</y>
     <width>256</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="purchase_price">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>150</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="sell_price">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>190</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="ticket_name">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>110</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="select_date">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>70</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="modify_label">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>20</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>最近添加项</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancel</sender>
   <signal>clicked()</signal>
   <receiver>Ticket_Sort_Manage</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>338</x>
     <y>263</y>
    </hint>
    <hint type="destinationlabel">
     <x>329</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>confirm</sender>
   <signal>clicked()</signal>
   <receiver>Ticket_Sort_Manage</receiver>
   <slot>on_ok()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>225</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>193</x>
     <y>153</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>purchase_compute_way</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>Ticket_Sort_Manage</receiver>
   <slot>on_purchase_price_compute_way_selected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>421</x>
     <y>153</y>
    </hint>
    <hint type="destinationlabel">
     <x>444</x>
     <y>67</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sell_compute_way</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>Ticket_Sort_Manage</receiver>
   <slot>on_sell_price_compute_way_selected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>401</x>
     <y>201</y>
    </hint>
    <hint type="destinationlabel">
     <x>452</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>select_date_btn</sender>
   <signal>clicked()</signal>
   <receiver>Ticket_Sort_Manage</receiver>
   <slot>on_ticket_add_date_selected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>125</x>
     <y>73</y>
    </hint>
    <hint type="destinationlabel">
     <x>148</x>
     <y>43</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>on_new_coal_added()</signal>
  <slot>on_ok()</slot>
  <slot>on_purchase_price_compute_way_selected()</slot>
  <slot>on_sell_price_compute_way_selected()</slot>
  <slot>on_ticket_add_date_selected()</slot>
 </slots>
</ui>
